### DISCUSSION TITLE
[Kavita] Dual Screen Reader Support / Improvements

### DISCUSSION THREAD

**Discussion by SScorpio00** [Kareadita/Kavita#3012] [Ideas]
### Idea Description

Could the other thread be removed. I sadly noticed I was logged in under a work account.

I found a more budget friendly dual screen laptop that I've started using as a comic reader. I've been slowly learning it as well as developing a solution to allow a spanned web browser across both screens which is working, but Kavita is presenting a few issues that might be issues for people just using the double layout viewing mode on a single screen.

Here's a gallery just showing Kavita in a web browser so you can see what I'm talking about. 
[https://imgur.com/a/tfLL8Wb](https://imgur.com/a/tfLL8Wb)

1. Double layout is set to just vertical space - Could it optionally scale based on fitting two images side by side instead?

The problem is the screens are just outside of 16x10. When scaled to vertically fix it's too wide for two images so only a single centered image is displayed. I have a custom program I'm developing to let a Chromium browser span the two screens, and I had to limit the vertical sizing so comics are displaying correctly, but different aspect ratios will cause this to break.

2. Could single page images be displayed on the left or right page rather than the middle?

Right now covers are displayed in the middle, being able to have it placed either as a left or right image in the double mode would let it display on a single screen.

3. Issues with the what is the starting page, could this be a selection?

Some comic files have multiple covers, or others don't include a scan of the inside of the front cover. This can break the display order. Would it be possible to add an option that would force the image displayed on in the left position, into the right? Basically inserting a blank empty page to reorder all of the others.

This may be related to this: https://github.com/Kareadita/Kavita/discussions/2750

### Idea Category

Web UI

### Duration of Using Kavita

About a year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by DieselTech** [comment]
> Could the other thread be removed

Done.

**Reply by majora2007** [reply]
This is another related (and much needed) FR:
https://github.com/Kareadita/Kavita/discussions/2660

**Reply by SScorpio00** [reply]
Yes, the paging number is the primary issue and breaks some comics being readable, especially if they do a two page splash as separate images or have a layout where you need to read across the top of two pages and then and follow the panels below.

I was able to work around the vertical sizing and centered single page in a reader app I developed.
https://github.com/SScorpio00/AllScreenReader
